mica/archive/aca_l0.py

```diff
@@ -163,10 +198,18 @@ def get_slot_data(start, stop, slot, imgsize=None,
         if 'FILENAME' in columns:
             all_rows['FILENAME'][rowcount:(rowcount + len(chunk))] = f['filename']
         if 'IMGRAW' in columns:
-            all_rows['IMGRAW'].reshape(rows, 8, 8)[
-                rowcount:(rowcount + len(chunk)), 0:f_imgsize, 0:f_imgsize] = (
-                chunk.field('IMGRAW').reshape(len(chunk),
-                                              f_imgsize, f_imgsize))
+            if img_shape_8x8:
+                if f_imgsize == 8:
+                    all_rows['IMGRAW'][rowcount:(rowcount + len(chunk))] = chunk.field('IMGRAW')
```

Comment:
As long as you are in here, maybe fix this many times repetition of `rowcount:(rowcount + len(chunk))`.  Define `idx0` and `idx1` up front and use it everywhere.  And isn't `chunk[name]` the same as `chunk.field(name)` these days?

Reply:
Will do. Indeed, I just read a fits file and `chunk[name]` the same as `chunk.field(name)`. No idea what it was in the old days 😃